frontend/public/index.html

```diff
@@ -64,5 +64,29 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <script>
+      if ('%NODE_ENV%' === 'production' && "serviceWorker" in navigator) {
+        window.addEventListener("load", function () {
+          navigator.serviceWorker
+            .register("./serviceWorker.js")
+            .then(
+              function (registration) {
+                console.log(
```

Comment:
nit: these can be removed now that you're done with your debugging. up to you :)